### DISCUSSION TITLE
IMAP and M365 account - i am confused

### DISCUSSION THREAD

**Discussion by Foxhunter-de** [frappe/helpdesk#2065] [Q&A]
Hi there,

I am new to Frappe (got a hosting at frappe cloud) and the helpdesk. I just want to add an IMAP account, but I can only choose between five or six services, like outlook.com, yahoo, etc. - I don't have any of them, I have my own hosting. I also want to add a Microsoft365 mail account. I can't find anything to help me. I am very confused because I think I am missing something. For a helpdesk it should be basic to add simple mail accounts. So what am I doing wrong?

Kind regards
Christian

**Comment by HachimanSec** [comment]
Hi Christian, 
same problem here. I don't use any of these options available and simply have to add a SMTP server with basic settings like:
Port: 587
Authentication method: PLAIN
Encryption: STARTTLS

**Reply by Foxhunter-de** [reply]
Hi,

thanks for the answer. As far as I know, sooner or later Microsoft will discontinue the default SMTP service. The correct way to set this up is to actually leave the helpdesk and go back to ERPNext and set up the email account there. It is very confusing to have multiple options and pages to set up email accounts - but within ERPNext you can select O-Auth for the Microsoft login and it will work. Overall I'm not happy, I'm a paying customer for Frappe and the support just doesn't give a f*ck about problems. My cloud instance loses connection to my mail service every few days and they just tell me it's my problem and won't even look at their logs. We are currently looking at alternatives, paying $50 a month and getting no support is just not OK.

**Reply by HachimanSec** [reply]
Hi, 
Thanks for your insights and sharing your experience. Not really convincing. I thought they would have a viable option compared to other vendors. 
I also asked them about the possibilities to get supported for a self hosted instance. They offered me a service for 1000 USD per month!?

As promising as the Frappe stack looked in the beginning, it might not be enterprise ready - by far.

**Reply by Foxhunter-de** [reply]
Hi,

today I made the decision to replace Frappe and ERPNext in my company. It just doesn't make sense anymore with the constant errors, the email addresses haven't been working properly again for 3 weeks. Tickets are not being created, replies are not being sent and support sometimes takes over a week to reply. Now there's also a bug where you can't send mails if someone is in the CC or BCC field - all in all, it's a big disaster for the money that's being charged. At the moment, in my opinion, it's a system for enthusiasts and hobbyists, but nothing for productive use in companies.
